.pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
```

Comment:
I think we are already using `flake8`. So, this would be just a duplicate thing to use for the same purpose.